src/collective/cover/tests/test_layout_resize.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+
+Resource  cover.robot
+Library  Remote  ${PLONE_URL}/RobotRemote
+
+Suite Setup  Open Test Browser
+Suite Teardown  Close all browsers
+
+*** Variables ***
+
+${basic_tile_name} =  "collective.cover.basic"
+${tile_class} =  div.cover-tile
+
+*** Test cases ***
+
+Test Columns-Titles and Resize
+
+    Enable Autologin as  Site Administrator
+    Go to Homepage
+
+    Create Cover  Save-Cover  Test-cover
+    Open Layout Tab
+
+    Add Tile  ${basic_tile_name}
+    ${data-column-size_before}=  Get Element Attribute  xpath=//*[@id="content"]/div[2]/div/div@data-column-size
```

Comment:
If you use `css=div .cover-column` will not work?

Another thing. Specifying the attribute in the locator is deprecated. See:

https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Get%20Element%20Attribute

Reply:
not, not work, I used something like this:

${var}= Get Element Attribute css=.class.class2.etc data(This is the attribute, which I want to get its value)

Reply:
What was the error?

Reply:
Get Element Attribute wait 1 argument but get 2.

That's why I opted for xpath